### DISCUSSION TITLE
Does Milvus support bulk insert using CURL?

### DISCUSSION THREAD

**Discussion by balloon1995** [milvus-io/milvus#24856] [Q&A and General discussion] [answered]
hi team,

Does milvus support bulk insert using curl？ We only found SDK API in ur official document:
https://milvus.io/docs/bulk_insert.md#Insert-entities-from-files

**Comment by yhmo** [comment, marked as answer]
Curl depends on rest api, but the Milvus rest api is out of date(the latest update is on milvus v2.1). So, it doesn't support bulkinsert using curl.
